db/products.js

```diff
@@ -0,0 +1,82 @@
+const productCollection = {
+  products: [],
+};
+let productId = 1;
+
+function create(obj) {
+  const hasKeys = obj.name && obj.price && obj.inventory;
+
+  if (hasKeys) {
+    obj.id = productId;
+    productCollection.products.push(obj);
+    productId++;
+    return `{ "success": true}`;
```

Comment:
This probably doesn't need to be a template literal since you're not evaluating any variables.